src/main/java/com/microsoft/azure/sample/controller/TodoListController.java

```diff
@@ -54,9 +51,13 @@ public TodoListController() {
     @RequestMapping(value = "/api/todolist", method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
     public ResponseEntity<?> getAllTodoItems() {
         try {
-            return new ResponseEntity<List<TodoItem>>(todoItemRepository.findAll(), HttpStatus.OK);
+            final Iterable<TodoItem> itemIterable = todoItemRepository.findAll();
+            final List<TodoItem> items = new ArrayList<>();
+            itemIterable.forEach(todoItem -> items.add(todoItem));

```

Comment:
Just curious why not returning the Iterable directly?

Reply:
This is an update according to API change, previously `findAll` returns `List`, while new version returns `Iterable`, just updated to remove the convert and the generics `List<TodoItem>` type.